spec/services/document_capture_session_result_spec.rb

```diff
@@ -64,6 +64,17 @@
         )
         expect(result.success?).to eq(true)
       end
+      it 'reports correctly from success when missing doc_auth_success and selfie_status' do
+        result = DocumentCaptureSessionResult.new(
+          id: id,
+          success: true,
+          pii: pii,
+          attention_with_barcode: false,
+          selfie_status: :nil,
```

Comment:
Nit: I think this should be 
`selfie_status: nil`
You could also just do 

```
  result = DocumentCaptureSessionResult.new(
          id: id,
          success: true,
          pii: pii,
          attention_with_barcode: false
   )
```
Both work. 🤷🏻‍♀️

Reply:
@eileen-nava removed.